spring-cloud-gcp-starters/spring-cloud-gcp-starter-sql/src/main/java/org/springframework/cloud/gcp/sql/GcpCloudSqlProperties.java

```diff
@@ -48,6 +48,8 @@
 
 	private Path credentialsLocation;
 
+	private String databaseType = "mysql";
```

Comment:
I'm pretty sure this can be of `DatabaseType` and, therefore, we don't need a custom `Validator` above.